go.mod

```diff
@@ -3,16 +3,16 @@ module github.com/elza2/go-cyclic
 go 1.18
```

Comment:
Whether the version needs to be updated go 1.21?

Reply:
my local toolchain is 1.21.4🥲.